crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -4789,6 +4767,37 @@ impl<'a> Checker<'a> {
         }
     }
 
+    /// Run any lint rules that operate over a single [`Binding`].
+    fn check_bindings(&mut self) {
```

Comment:
There should be a few others that we can move to this phase.

Reply:
We need to be careful with the term *phase* to avoid confusion. If `check_bindings` is a phase, then `check_deferred_scoipes` and others are too. But what I understand is that these are not considered phases (at least, we only documented 4).